airflow/gcp/example_dags/example_gcs.py

```diff
@@ -127,7 +128,16 @@
         task_id="delete_files", bucket_name=BUCKET_1, objects=[BUCKET_FILE_LOCATION]
     )
 
+    # [START howto_operator_gcs_delete_bucket]
+    delete_bucket = GCSDeleteBucketOperator(task_id="delete_bucket", bucket_name=BUCKET_1)
+    # [END howto_operator_gcs_delete_bucket]
+
     [create_bucket1, create_bucket2] >> list_buckets >> list_buckets_result
     [create_bucket1, create_bucket2] >> upload_file
     upload_file >> [download_file, copy_file]
     upload_file >> gcs_bucket_create_acl_entry_task >> gcs_object_create_acl_entry_task >> delete_files
+    create_bucket1 >> delete_bucket
```

Comment:
Does it ensure that bucket will be deleted after other tasks on it will finish? WDYT?

Reply:
Can you attach diagram? https://airflow.readthedocs.io/en/latest/usage-cli.html#exporting-dags-structure-to-images

Reply:
Yes, such situation causes errors. 
We had a chat about it today with Kamil, seems to be ok right know. I will commit the solution.